torchutils/models.py

```diff
@@ -1,16 +1,15 @@
 from ._validate import _validate_param
 
-__all__ = ['get_model_param_count']
+__all__ = ['get_model_param_count', 'get_train_param_count']
+
 
 def get_model_param_count(model):
```

Comment:
It will be better to merge the new functionality in the existing API with a new optional parameter. Perhaps we can improve the API to return total params, trainable params and non-trainable params as well.